app/models/step_by_step_page.rb

```diff
@@ -6,6 +6,8 @@ class StepByStepPage < ApplicationRecord
     submitted_for_2i
   ).freeze
 
+  enum scheduled_update_type: %i[major minor]
```

Comment:
Rails [`enum` feature](https://api.rubyonrails.org/v5.2.3/classes/ActiveRecord/Enum.html) stores the values as integers (giving the efficiency gains of integers, e.g. lower disk usage and faster querying) whilst allowing you to refer to them as a string throughout the application (as if they were a string).  This would probably be a good thing to have for the `status` column too.